### ISSUE TITLE
Example code for client doesn't work

### ISSUE THREAD

**Issue by flip111** [jquast/telnetlib3#27]
Python 3.6.3

```
Traceback (most recent call last):
  File "/home/flip111/prj/venv/bin/adev", line 11, in <module>
    sys.exit(cli())
  File "/home/flip111/prj/venv/lib64/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/flip111/prj/venv/lib64/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/flip111/prj/venv/lib64/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/flip111/prj/venv/lib64/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/flip111/prj/venv/lib64/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/flip111/prj/venv/lib64/python3.6/site-packages/aiohttp_devtools/cli.py", line 88, in runserver
    run_app(*_runserver(**active_config))
  File "/home/flip111/prj/venv/lib64/python3.6/site-packages/aiohttp_devtools/runserver/main.py", line 46, in runserver
    config.check()
  File "/home/flip111/prj/venv/lib64/python3.6/site-packages/aiohttp_devtools/runserver/config.py", line 171, in check
    app_factory = self.import_app_factory()
  File "/home/flip111/prj/venv/lib64/python3.6/site-packages/aiohttp_devtools/runserver/config.py", line 138, in import_app_factory
    module = import_module(module_path)
  File "/usr/lib64/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/flip111/prj/src/server/app.py", line 106, in <module>
    loop.run_until_complete(reader.protocol.waiter_closed)
AttributeError: 'TelnetReaderUnicode' object has no attribute 'protocol'
```

was also wondering why there is `buf = ''` in the example code

**Comment by jquast** [comment]
closed in 0b75f8c83d5b3122d330ce4676dc21dbcb85d74c thanks

**Comment by flip111** [comment]
Hey thanks for the fix. Just a question .. why is there `while True:` in the example code? Now i have a loop in async code that is already in the event loop?!
